recap/converters/postgresql.py

```diff
@@ -144,7 +145,7 @@ def _parse_type(self, column_props: dict[str, Any]) -> RecapType:
                 symbols=column_props["ENUM_VALUES"],
             )
         else:
-            raise ValueError(f"Unknown data type: {data_type}")
+            base_type = UnknownType()
```

Comment:
Incldue `base_type` in unknown type so devs can debug? Any other useful info, too...